### PR TITLE
Call xspec from inside igmguesses

### PR DESCRIPTION
**Pull request by profxj**
Adds keystroke "&" to enable the igmguesses user to launch
a new widget (a modified xspec).

It also converts the list of componetns into AbsSystems
and each of these is plotted (with unique color coding).

No tests or docs (aside from top doc string).

It may make sense to push the new Dialog into linetools.
Don't merge until we decide on that.

### CODE REVIEW

**Comment by ntejos** [comment]
What would be the main idea of having xspec loaded within IGMGuesses?

**Comment by profxj** [comment]
1. View the full spectrum vs. wavelength
1. Inspect which lines are still not identified (and which are)

File: pyigm/guis/igmguesses.py [hunk 1]
```diff
@@ -1836,4 +1854,69 @@ def from_igmguesses_to_joebvp(infile, outfile):
...
+            z = event.xdata/wrest.value - 1.
+            self.pltline_widg.llist['z'] = z
+
+            self.pltline_widg.zbox.setText('{:.5f}'.format(self.pltline_widg.llist['z']))
+
+            # Draw
+            self.spec_widg.on_draw()
+
+    def quit(self):
+        self.close()
```
Comment (ntejos) [inline pyigm/guis/igmguesses.py]: As discussed, may be better to move this ExSpecDialog to linetools.

File: pyigm/guis/igmguesses.py [hunk 2]
```diff
@@ -1686,13 +1704,17 @@ def comp_init_attrib(comp):
 def sync_comp_lines(comp, only_lims=False):
     """Synchronize attributes of the lines and updates
     """
+    if only_lims:
+        # Need to update zlim *not* vlim
+        comp_vlim_mks = comp.vlim.to('km/s').value
+        zlim_comp = comp.zcomp + (1 + comp.zcomp) * (comp_vlim_mks / c_kms)
```
Comment (ntejos) [inline pyigm/guis/igmguesses.py]: Use `ltu.z_from_dv()` instead

**Review by ntejos — CHANGES_REQUESTED** [review]
Looks good, please merge after addressing the comment.

File: pyigm/guis/igmguesses.py (hunk 2 above)
Comment (profxj) [reply pyigm/guis/igmguesses.py]: replaced as requested.

**Comment by profxj** [comment]
Merging

**Pull request merged**